dts/bindings/sensor/nordic,nrf-qdec.yaml

```diff
@@ -0,0 +1,74 @@
+#
+# Copyright (c) 2018, Nordic Semiconductor ASA
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+---
+title: Nordic nRF Family QDEC node
+id: nordic,nrf-qdec
+version: 0.1
+
+description: >
+    This is a representation of the Nordic nRF QDEC node
+
+properties:
+    compatible:
+      type: string
+      category: required
+      description: compatible strings
+      constraint: "nordic,nrf-qdec"
+
+    reg:
+      type: array
+      description: mmio register space
+      generation: define
+      category: required
+
+    interrupts:
+      type: array
+      category: required
+      description: required interrupts
+      generation: define
+
+    a-pin:
+      type: int
+      description: A pin
+      generation: define
+      category: required
+
+    b-pin:
+      type: int
+      description: B pin
+      generation: define
+      category: required
+
+    led-pin:
+      type: int
+      description: LED pin for light based QDEC device (0xFFFFFFFF if unused)
```

Comment:
Instead of 0xFFFFFFFF if unused, make the category optional.

Reply:
Fixed